data/World/Overworld.json

```diff
@@ -74,7 +74,6 @@
     },
     {
         "region_name": "Kokiri Forest",
-        "font_color": "Green",
```

Comment:
Looks like this is reverting some of the changes from #1428. I'm guessing this is accidental?